compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/scenario/Conversions.scala

```diff
@@ -553,10 +553,11 @@ final class Conversions(
       .build()
   }
 
-  def convertNode(
-      nodeWithId: (NodeId, N.GenNode[NodeId, V.ContractId])
-  ): proto.Node = {
+  def convertIncompleteNode(
```

Comment:
renamed to avoid ad-hoc overloading with another function of the same name in this file.